bundles/core/org.eclipse.smarthome.core.thing.xml/src/main/java/org/eclipse/smarthome/core/thing/xml/internal/EventDescriptionConverter.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2014 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.thing.xml.internal;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.eclipse.smarthome.config.xml.util.GenericUnmarshaller;
+import org.eclipse.smarthome.config.xml.util.NodeIterator;
+import org.eclipse.smarthome.config.xml.util.NodeList;
+import org.eclipse.smarthome.config.xml.util.NodeValue;
+import org.eclipse.smarthome.core.types.EventDescription;
+import org.eclipse.smarthome.core.types.StateOption;
+
+import com.thoughtworks.xstream.converters.ConversionException;
+import com.thoughtworks.xstream.converters.Converter;
+import com.thoughtworks.xstream.converters.UnmarshallingContext;
+import com.thoughtworks.xstream.io.HierarchicalStreamReader;
+
+/**
+ * The {@link EventDescriptionConverter} is a concrete implementation of the {@code XStream} {@link Converter} interface
+ * used to convert a event description within an XML document
+ * into a {@link EventDescription} object.
```

Comment:
That's a bug in the ESH formatter options in Eclipse, which are set up through the Eclipse Installer. Fixed it nonetheless.